### PR TITLE
splitting up decrypter.js into multiple sub files

### PR DESCRIPTION
**Pull request by BrandonOCasey**

### CODE REVIEW

File: src/decrypter/async-stream.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import Stream from '../stream';
```
Comment (dmlap) [inline src/decrypter/async-stream.js]: This file needs a comment describing its contents.

**Comment by dmlap** [comment]
My goals for modules in this project are:

- They are short enough you can read through them easily. 500 lines or so seems like a good target.
- They are not so short it is a pain to figure out how things fit together. Less than 20 lines seems like a warning sign to me.
- They export functionality that is well-defined and useful stand-alone.
- The functionality they export is actually used in the app.

The decrypt function and object, and the ntoh function don't to meet those criteria for me. AsyncStream is designed to be generic enough that it *might* be useful outside the decryption process so extracting that is fine by me. The other are not useful enough independently to be worth extracting and the combined module is still reasonably sized.

File: src/decrypter/decrypter.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
+import AES from './aes';
```
Comment (dmlap) [inline src/decrypter/decrypter.js]: Needs a file-level comment.

File: src/decrypter/async-stream.js (hunk 1 above)
Comment (BrandonOCasey) [inline src/decrypter/async-stream.js]: added

File: src/decrypter/decrypter.js (hunk 2 above)
Comment (BrandonOCasey) [inline src/decrypter/decrypter.js]: added

**Pull request merged**